compiler/natives/src/crypto/rand/rand.go

```diff
@@ -5,6 +5,7 @@ package rand
 import (
 	"errors"
 
+	"github.com/gopherjs/gopherjs/compiler/natives/src/crypto/webcrypto"
```

Comment:
I did the move: https://github.com/gopherjs/gopherjs/pull/558/commits/02e03500ee7cd665b1dfad6b47240e4dc808f46a